client/components/product-card/index.jsx

```diff
@@ -76,7 +76,7 @@ const ProductCard = ( {
 ProductCard.propTypes = {
 	billingTimeFrame: PropTypes.string,
 	currencyCode: PropTypes.string,
-	description: PropTypes.oneOfType( [ PropTypes.string, PropTypes.element ] ),
+	description: PropTypes.oneOfType( [ PropTypes.string, PropTypes.element, PropTypes.node ] ),
```

Comment:
Does the `PropTypes.element` still make sense here?

Reply:
Yeah, we should leave it as it might be useful for any additional consumers of this block.